File: Video Shoppe/Customer.cpp
#include "Customer.h"

using namespace std;

Customer::Customer() {

}

Customer::~Customer() {
	cout << "Customer removed" << endl;
}

DVD Customer::getRentals() {
	return *Customer::rentals;
}

void Customer::updateAddress(Address newAddress) {
	Customer::address = newAddress;
}

void Customer::updatePhone(string newPhone) {
	Customer::phone = newPhone;
}

void Customer::updateBankCards(BankCard newCard) {
	for (int i = 0; i < sizeof(Customer::bankCards); i++) {
		if (Customer::bankCards[i] != nullptr)
			continue;
		else
			Customer::bankCards[i] = newCard;
		return;
	}
}

File: src/business/Customer.java
package business;

import java.sql.Date;

public class Customer {
	public String customerID;
	private String lastName;
	private String firstName;
	private char middleInitial;
	private Date birthDate;
	private int age;
	private String address;
	private String phone;
	private String bankCard;
	private DVD[] rentals;
	
	public Customer() {
		
	}
	
	public DVD[] getRentals() {
		return rentals;
	}
	
	public void updateAddress(String newAddress) {
		address = newAddress;
	}
	
	public void updatePhone(String newPhone) {
		phone = newPhone;
	}
	
	public void updateBankCard(String newCard) {
		bankCard = newCard;
	}
}
